module/commands/esami.py

```diff
@@ -101,7 +101,7 @@ def esami_button_anno(update: Update, context: CallbackContext, chat_id: int, me
         chat_id: id of the chat of the user
         message_id: id of the sub-menu message
     """
-    locale: str = update.message.from_user.language_code
+    locale: str = update.message.from_user.language_code if update.message.from_user.language_code else update.from_user.language_code
```

Comment:
wrap this into a function instead of repeating 3 times a big long string please 🙏